safe/test/data/scenario/scenario_1.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
The file name. "scenario_1.json" is not very explicit, if we have many scenarii, then it's gona be difficult to know.